### PR TITLE
[DC-377] Fix a few typos in workspace script

### PR DESCRIPTION
**Pull request by s-rubenstein**

### CODE REVIEW

File: scripts/workspace-conversion/workspace-to-dataset.py [hunk 1]
```diff
@@ -298,7 +298,6 @@ def generate_catalog_metadata(workspace):
         "dct:title": wsAttributes.get("library:datasetName", None),
         "dct:version": wsAttributes.get("library:datasetVersion", None),
         "dct:description": wsAttributes.get("library:datasetDescription", None),
-        "TerraDCAT_ap:hasOwner": wsAttributes.get("library:datasetOwner", None),
```
Comment (s-rubenstein) [inline scripts/workspace-conversion/workspace-to-dataset.py:301]: This was duplicated on line 312

**Review by fboulnois — APPROVED** [review]

File: scripts/workspace-conversion/workspace-to-dataset.py (hunk 1 above)
Comment (fboulnois) [reply scripts/workspace-conversion/workspace-to-dataset.py:301]: nice catch!

**Review by pshapiro4broad — APPROVED** [review]

File: scripts/workspace-conversion/workspace-to-dataset.py [hunk 2]
```diff
@@ -331,20 +335,11 @@ def main():
...
     print("JSON Result:\n", json.dumps(metadata))
     print("------------------------------------------------------------")
-    print("Demo Link:")
-    # Add demo fields, like the server would
-    metadata["id"] = f"external_id_{uuid.uuid4()}"
-    metadata["dct:modified"] = time.time()
-    metadata["roles"] = ["reader", "admin "]
-    jsonMetadata = json.dumps(metadata)
-    print(
-        f"{terraUiRoot}/#library/browser?externalWorkspace={urllib.parse.quote(jsonMetadata)}"
```
Comment (s-rubenstein) [inline scripts/workspace-conversion/workspace-to-dataset.py:345]: This demo URL isn't merged into Terra UI so I removed it.

File: scripts/workspace-conversion/workspace-to-dataset.py [hunk 3]
```diff
@@ -331,20 +335,11 @@ def main():
...
-    )
+    print("Unused Attributes \n", json.dumps(unusedWorkspaceAttributes))
```
Comment (s-rubenstein) [inline scripts/workspace-conversion/workspace-to-dataset.py:342]: This is something that Phil and I talked about maybe being useful from the script for debugging.

File: scripts/workspace-conversion/workspace-to-dataset.py [hunk 4]
```diff
@@ -309,18 +310,21 @@ def generate_catalog_metadata(workspace):
...
         "prov:wasAssociatedWith": next(
-            iter(wsAttributes.get("library:institute", {}).get("items", [])), None
+            iter(wsAttributes.pop("library:institute", {}).get("items", [])), None
         ),
         "prov:wasGeneratedBy": get_workspace_generated(wsAttributes),
         "files": get_workspace_files(wsAttributes),
-        "TerraDCAT_ap:hasConsentGroup": wsAttributes.get("library:orsp", None),
+        "TerraDCAT_ap:hasConsentGroup": wsAttributes.pop("library:orsp", None),
+        "workspaces": {
+            "legacy": workspace
```
Comment (s-rubenstein) [inline scripts/workspace-conversion/workspace-to-dataset.py:323]: This was missing and was in the "workspaces in TIM" doc.

File: scripts/workspace-conversion/workspace-to-dataset.py [hunk 5]
```diff
@@ -226,9 +224,9 @@ def get_workspace_contributors(wsAttributes):
         if "library:datasetDepositor" in wsAttributes:
             contributor["contactName"] = wsAttributes["library:datasetDepositor"]
             contributor["correspondingContributor"] = True
-        contributor["email"] = wsAttributes.get("library:contactEmail", None)
-        contributor["intstitution"] = next(
-            iter(wsAttributes.get("library:institute", {}).get("items", [])), None
+        contributor["email"] = wsAttributes.pop("library:contactEmail", None)
```
Comment (s-rubenstein) [inline scripts/workspace-conversion/workspace-to-dataset.py:227]: The move to pop is to support unused attributes.

Comment (fboulnois) [reply scripts/workspace-conversion/workspace-to-dataset.py:228]: If I am understanding correctly, you replace the use of `get` with `pop` so you can see what remains in (the "unused attributes") and show it to the user. Is that correct?

**Pull request merged**